codalab/lib/codalab_manager.py

```diff
@@ -530,7 +530,7 @@ def check_version(self, server_version):
         epoch_str = formatting.datetime_str(datetime.datetime.utcfromtimestamp(0))
         last_check_str = self.state.get('last_check_version_datetime', epoch_str)
```

Comment:
We probably also need to check what's the timezone when generating `last_check_version_datetime`, as the calculation in the next two lines will depend on this.